### DISCUSSION TITLE
What GHG Emission factors do you use for this tool? from here https://cfpub.epa.gov/si/si_public_record_Report.cfm?dirEntryId=349324&Lab=CESER

### DISCUSSION THREAD

**Discussion by andiosika** [USEPA/useeior#287] [Q&A] [answered]
Trying to verify emission factors and map them to a version found here https://cfpub.epa.gov/si/si_public_record_Report.cfm?dirEntryId=349324&Lab=CESER can you point me to the version number?

**Comment by bl-young** [comment, marked as answer]
The [versioning](https://github.com/USEPA/supply-chain-factors/blob/main/Versioning.yml) of the supply chain factors indicates the minimum version of useeior required to generate them. More recent versions should still work (that is the latest release of useeior can be used to generate the supply chain factors). The latest model spec for those factors is here: https://github.com/USEPA/supply-chain-factors/blob/main/model-specs/USEEIOv2.1.19-GHG.yml
